cluster/charts/rook-ceph/templates/resources-k8s-1.15.yaml

```diff
@@ -0,0 +1,735 @@
+---
```

Comment:
The contents of this file will need to be conditionally installed. The helm integration test is failing because both versions of the CRDs are being installed.

Ideally we would find a way to conditionally install the correct CRDs based on the K8s version. At a glance, I don't see in the helm docs how to do this. 

If that isn't possible, we could fall back to use a condition like this where we define a new value in values.yaml.
```
{{- if not .Values.enableV1Extensions }}
...
{{- end }}
```

Reply:
As we discussed, I found this to compare with k8s version `{{- if semverCompare "<=1.15.0" .Capabilities.KubeVersion.GitVersion }}`

Reply:
In the v1.15 CI, the CRDs are not being installed. 
```
Error from server (NotFound): error when creating "STDIN": the server could not find the requested resource (post cephclusters.ceph.rook.io)
```
I see that the condition in resources.yaml worked fine for >= 1.16 and the syntax looks the same, so not sure what's wrong here...

Reply:
If the condition is working in `resources.yaml` for >=1.16, I placed `yaml` for >=1.15  inside `else`. Let's see if it works.